content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Views'
+metaTitle: 'How to include views in your Prisma schema'
+metaDescription: 'How to include views in your Prisma schema'
+hidePage: false
+preview: true
+tocDepth: 2
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+Support for views is currently a very early, experimental preview feature. You can add a view to your Prisma schema with the `view` keyword, but you cannot introspect your schema with `db pull` or apply your schema to your database with Prisma Migrate and `db push`.<br /><br />For updates on progress with this feature, follow [our GitHub issue](https://github.com/prisma/prisma/issues/17335).
```

Comment:
```suggestion
Support for views is currently a very early preview feature. You can add a view to your Prisma schema with the `view` keyword, but you cannot introspect your schema with `db pull` or apply your schema to your database with Prisma Migrate and `db push`.<br /><br />For updates on progress with this feature, follow [our GitHub issue](https://github.com/prisma/prisma/issues/17335).
```

`very early, experimental preview feature.` it's a weird maturity level
https://www.prisma.io/docs/about/prisma/releases#product-maturity-levels

In the past, we got rid of experimental
Now we have early access and preview 
This mixes 3 different things 🔥

Reply:
Later in this file we say `Support for views is currently in an early preview.`.
So let's remove experimental at least?

Reply:
This is the wording we used for multi-schema, but yes maybe it would be better to remove 'experimental'. Will have a think about how to reword

Reply:
"very early preview feature" is possibly enough, but I feel that "experimental" highlights a bit more that things are really early. Both works for me.

Reply:
I've removed 'experimental'